rpg/load_game_map.py

```diff
@@ -129,9 +130,9 @@ def load_map(map_name):
     for layer, sprite_list in game_map.map_layers.items():
         if "_blocking" in layer and not GOD_MODE:
             try:
-              game_map.scene.remove_sprite_list_by_object(sprite_list)
+                game_map.scene.remove_sprite_list_by_object(sprite_list)
             except:
-              print(f'{layer} has no objects')
+                print(f'{layer} has no objects')
```

Comment:
We should also consider replacing all print statements with either a logging library like `loguru` or rich's `console.print`. We should abstract a global STDOUT module where we can import our own version of `print()` or `log.<>()`

Reply:
Yup good idea for future refactor!